src/components/LayoutTree.tsx

```diff
@@ -707,6 +737,9 @@ const LayoutTree = () => {
                   // Do this as padding instead of y, otherwise there will be a gap between drop targets.
                   style={cliff < 0 ? cliffPaddingStyle : undefined}
                   crossContextualKey={key}
+                  prevCliff={prevCliff}
```

Comment:
This should just be inlined as `treeThoughtsPositioned[index - 1]?.cliff`. It's short and readable and only used in one place, so it doesn't need to be extracted into a variable.